app/scripts/controllers/push-platform-notifications/utils/get-notification-image.ts

```diff
@@ -0,0 +1,6 @@
+import browser from 'webextension-polyfill';
+
+export async function getNotificationImage() {
+  const iconUrl = await browser.runtime.getURL('../../images/icon-64.png');
```

Comment:
This correctly gets the browser image (used on other notifications - e.g. transaction notifications)